test/maps/lua_testsuite.wmf/scripting/tribes_descriptions.lua

```diff
@@ -51,13 +51,13 @@ function test_descr:test_get_builder()
    assert_equal("atlanteans_builder", tribe.builder)
 end
 
-function test_descr:test_get_carrier()
+function test_descr:test_get_carriers()
    local tribe = egbase:get_tribe_description("atlanteans")
+   assert_equal("atlanteans_carrier", tribe.carriers[1])
+   assert_equal("atlanteans_horse", tribe.carriers[2])
+   assert_equal(2, #tribe.carriers)
+   -- deprecated, test until removed:
```

Comment:
What we mean by "deprecated" in the documentation is that `carriers` is now preferred but `carrier` and `carrier2` will continue to work for API/add-ons compatibility. No removal of deprecated functionality is planned unless absolutely unavoidable.
```suggestion
   -- Deprecated, but shall still work for API compatibility:
```